examples/monosynth_basic/sketch.js

```diff
@@ -0,0 +1,20 @@
+/**
+ *  Play a random note
+ *  every time you press a key
+ */
+
+var monoSynth;
+
+function setup() {
+  monoSynth = new p5.MonoSynth();
+}
+
+function keyPressed() {
+  // pick a random midi note
```

Comment:
maybe we should use mouse because it'll then be accessible on touch devices, and because keyPressed will fire more than once

via https://p5js.org/reference/#/p5/keyPressed:

> Because of how operating systems handle key repeats, holding down a key may cause multiple calls to keyTyped() (and keyReleased() as well). The rate of repeat is set by the operating system and how each computer is configured.